test/end-to-end/core.sh

```diff
@@ -566,7 +566,7 @@ os::log::info "Validating image pruning"
 # builder service account should have the power to create new image streams: prune in this case
 os::cmd::expect_success "docker login -u e2e-user -p $(oc sa get-token builder -n cache) -e builder@openshift.com ${DOCKER_REGISTRY}"
 os::cmd::expect_success 'docker pull busybox'
-GCR_PAUSE_IMAGE="gcr.io/google_containers/pause:3.0"
+GCR_PAUSE_IMAGE="openshift/origin-pod:v3.6.0"
```

Comment:
>  :latest might not be always on local storage

Images are a hard pre-requisite for `hack/test-end-to-end.sh` so we can be certain they will have been built before this test runs. They are built and tagged for the SHA we're testing and `:latest` always. I think it is safe to assume